config/samples/lws_v1alpha1_configuration.yaml

```diff
@@ -1,12 +1,33 @@
 apiVersion: lws.x-k8s.io/v1alpha1
 kind: Configuration
 metadata:
-  labels:
```

Comment:
Any particular reason to remove them?

Reply:
this is an example configuration to show how to configure LWS global setup via this CRD.
As an example, I did not find  those label managed by helm or kustomize do anything real help ?
did I missing anything ? @kerthcet